src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -410,13 +463,23 @@ def load_arguments(self, _):
         c.argument('crg_id', validator=validate_crg_id, is_preview=True)
         # no validation for aks create because it already only supports Linux.
         c.argument('message_of_the_day')
-        c.argument('workload_runtime', arg_type=get_enum_type(workload_runtimes), default=CONST_WORKLOAD_RUNTIME_OCI_CONTAINER)
+        c.argument('workload_runtime', arg_type=get_enum_type(
+            workload_runtimes), default=CONST_WORKLOAD_RUNTIME_OCI_CONTAINER)
         # no validation for aks create because it already only supports Linux.
         c.argument('enable_custom_ca_trust', action='store_true')
-        c.argument('nodepool_allowed_host_ports', validator=validate_allowed_host_ports, is_preview=True, help="allowed host ports for agentpool")
-        c.argument('nodepool_asg_ids', validator=validate_application_security_groups, is_preview=True, help="application security groups for agentpool")
+        c.argument('nodepool_allowed_host_ports', validator=validate_allowed_host_ports,
+                   is_preview=True, help="allowed host ports for agentpool")
+        c.argument('nodepool_asg_ids', validator=validate_application_security_groups,
+                   is_preview=True, help="application security groups for agentpool")
         c.argument('node_public_ip_tags', arg_type=tags_type, validator=validate_node_public_ip_tags,
                    help='space-separated tags: key[=value] [key[=value] ...].')
+        c.argument('guardrails_level', arg_type=get_enum_type(guardrails_levels),
+                   help='The guardrails level, one of ["Off", "Warning", "Enforcement"]', is_preview=True)
+        c.argument('guardrails_version', type=str,
```

Comment:
Could omit this `type=str` as it's the default type.

Besides, may remove the help message here as you've written detailed help message in `_help.py`, and I think one of them will be overwritten.

Reply:
removed